examples/plain-javascript/README.md

```diff
@@ -30,10 +30,13 @@ Then, you'll get this:
 .
 ├── cldrjs
 │   ├── cldr.js
+│   ├── node_main.js
 │   └── cldr
 │       ├── event.js
 │       └── supplemental.js
-└── index.html
+│       └── unresolved.js
+├── index.html
+└── README.md
```

Comment:
I got your point that it's confusing for the user to see a different tree (he'll get more files than what's listed here). But, I also believe this tree should display the very minimal needed information.

How about we use dots "..." to represent that "other" files are there. Similarly of what we do in the `amd-bower` demo:
https://github.com/johnnyreilly/globalize/blame/master/examples/amd-bower/README.md#L42